install/container/centos.sh

```diff
@@ -4,11 +4,30 @@
 
 set -e
 
-# INCLUDE+ common/Dockerfile.yum-install-required
 yum update -y && yum makecache \
- && yum install -y systemd procps-ng sudo shadow-utils passwd git libnsl sudo ca-certificates openssh-server openssh-clients util-linux-ng util-linux-user libuser python3 cmake3 \
- && yum groupinstall -y 'Development Tools' \
- && yum install -y perl-core perl-IPC-Cmd # OpenSSL 3 build system requires this
+  && yum install -y openssh-server \
+  sudo \
+  unzip \
+  git
+
+if [ "$TIPI_INSTALL_LEGACY_PACKAGES" = "ON" ]; then
+ # INCLUDE+ common/Dockerfile.yum-install-required
+  yum update -y && yum makecache \
+    && yum install -y systemd \
+    procps-ng \
+    shadow-utils \
+    passwd \
```

Comment:
I supposed we need generally shadow-utils passwd no if it is here, same for procps-mg (tipi remote_serve checks) an